packages/stryker-vue-mutator/src/helpers/MutatorHelpers.ts

```diff
@@ -0,0 +1,15 @@
+import { Mutator, MutatorFactory } from 'stryker-api/mutant';
+import { Config } from 'stryker-api/config';
+
+const generateMutators = (config: Config): { [name: string]: Mutator; } => {
```

Comment:
This name... In what way is it actually generating mutators? More like `discoverMutators` right?